### DISCUSSION TITLE
Blog Post [2]

### DISCUSSION THREAD

**Discussion by gordi76** [NicoReimer/Breakpoint#8] [Weekly-Blog]
Diese Woche haben wir unsere Software Requirements Specification (<a href="https://github.com/NicoReimer/Breakpoint/blob/main/SRS-Wiki-01.pdf">SRS</a>) erstellt.

**Comment by Yahya-EzEd** [comment]
Hallo Zusammen 😃,

wir haben das Dokument überflogen und das dabei festgestellt:
- Die Anforderungen zeigen klar den Mehrwert für die Nutzer.
- Die Dokumentation ist gut strukturiert und verständlich.
- Anforderungen sind größtenteils präzise formuliert.
- Logische Kohärenz ist gegeben.
- Realistisch - Ressourcen und Umsetzungsaufwand erscheinen angemessen.
- Die meisten Anforderungen sind klar messbar.
- Die angestrebte Systemqualität ist gut beschrieben.

Insgesamt wirkt das Konzept durchdacht und gut dokumentiert 👍.

**Comment by stefanschfr** [comment]
Hallo,

Euer SRS ist gut strukturiert und verständlich, besonders die klaren Anforderungen und der Fokus auf Usability und Accessibility sind gelungen.

Ein kleiner Hinweis: Sowohl in der Einleitung als auch im Abschnitt „Purpose“ wird der Zweck des SRS beschrieben. Ihr könnt überlegen, den Teil in der Einleitung, der sich auf den Zweck des SRS bezieht, in den Purpose zu übernehmen, um Wiederholungen zu vermeiden.

Insgesamt eine tolle Arbeit – ich freue mich auf das fertige Produkt!

Viele Grüße

Sebastian Albert, Oskar, Luca, Philip und Stefan
